src/Configuration/ActionConfigRowDefinition.php

```diff
@@ -27,10 +27,10 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->isRequired()
                     ->cannotBeEmpty()
                 ->end()
-                ->append($this->addDbNode())
+                ->append(DbNode::create())
```

Comment:
I think this should be dependency injected so that it can be easily replaced in concrete extractors... 

```
new ActionConfigRowDefinition(new DbNode(new SshNode()));
// in weird db ex
new ActionConfigRowDefinition(new MyWeirdDbNodeWithoutHostname(new SshNode()));
```